### PR TITLE
Chapter 9: Advanced login

### PR DESCRIPTION
**Pull request by lanlehoang67**
image

### CODE REVIEW

**Comment by lanlehoang67** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -5,15 +5,16 @@ def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate params[:session][:password]
       log_in user
+      params[:session][:remember_me] == "1" ? remember(user) : forget(user)
```
Comment (hhhhuy112) [inline app/controllers/sessions_controller.rb]: "1" cho vào file settings.yml em

**Comment by hhhhuy112** [comment]
commented

**Comment by lanlehoang67** [comment]
ready

**Review by hhhhuy112 — APPROVED** [review]

**Comment by hhhhuy112** [comment]
merged

**Pull request merged**